### PR TITLE
Show image thumbnails in layout views - 4884

### PR DESCRIPTION
**Pull request by michaelrogers**
Closes  #4884 

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [x] Unit tests included and/or updated with changes?
* [x] Command line build passes?
* [ ] Has this been smoke tested?
* [x] Testing instructions included in associated issue?

### Reviewer Checklist

* [x] Changes appear to address issue?
* [x] Changes appear not to be breaking changes?
* [x] Appropriate unit tests included?
* [x] Code style and in-line documentation are appropriate?
* [x] Commit messages meet standards?
* [x] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
* [x] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)

### CODE REVIEW

File: src/plugins/imagery/components/ImageryView.vue [hunk 1]
```diff
@@ -232,6 +233,7 @@ const ARROW_LEFT = 37;
 const SCROLL_LATENCY = 250;
 
 const ZOOM_SCALE_DEFAULT = 1;
+const SHOW_THUMBS_THRESHOLD_HEIGHT = 400;
```
Comment (akhenry) [inline src/plugins/imagery/components/ImageryView.vue:236]: @charlesh88 Does this seem like a reasonable threshold for showing the thumbnails? Would be great to get your feedback.

**Comment by akhenry** [comment]
@michaelrogers Is this otherwise good to go? Have you smoke tested?

**Comment by michaelrogers** [comment]
@akhenry - The functionality of this seems good to go. I've created a separate PR to add a `forceShowThumbnail` property on the state to fix a failing spec test. One minor functional change I made is instead of hiding via `style.display = 'none'` to a v-if selective render for a potential performance improvement if there is a sizeable number of telemetry items.
https://github.com/nasa/openmct/pull/5100

File: src/plugins/imagery/components/ImageryView.vue (hunk 1 above)
Comment (charlesh88) [reply src/plugins/imagery/components/ImageryView.vue:236]: Taking a look now...

**Comment by akhenry** [comment]
> @akhenry - The functionality of this seems good to go. I've created a separate PR to add a `forceShowThumbnail` property on the state to fix a failing spec test. One minor functional change I made is instead of hiding via `style.display = 'none'` to a v-if selective render for a potential performance improvement if there is a sizeable number of telemetry items. #5100

Great idea, thanks. This makes me realize that we are also requesting all those historical images unnecessarily in this case 😮 Future optimization.

**Comment by charlesh88** [comment]
### @charlesh88 change overview
- Thumbs now display a small version when height > 200px and <= 600px, modded current threshold and added new threshold accordingly;
- Timestamp added to thumbnails as a title attribute, allowing tooltips;
- Small thumbs don't display timestamps;
- Tweaks to thumb scrolling container height for better clearance against scrollbar;
- Removed hiliting of latest thumbnail to avoid confusion with selection context;

**Height <= 200px high, no thumbs displayed**

Screen Shot 2022-04-22 at 11 33 39 AM

**Height > 200px && height <= 600px, show small thumbs**

Screen Shot 2022-04-22 at 11 33 48 AM

**Height > 600px, show fullsize thumbs**

Screen Shot 2022-04-22 at 11 48 34 AM

**In Display Layout, showing tooltip on hover**

Screen Shot 2022-04-22 at 11 35 52 AM

**Comment by akhenry** [comment]
This gets two _thumbs_ up! 👍 👍

**Pull request merged**